contracts/optimistic-ethereum/OVM/execution/OVM_ExecutionManager.sol

```diff
@@ -158,7 +158,9 @@ contract OVM_ExecutionManager is iOVM_ExecutionManager, Lib_AddressResolver {
         override
         public
     {
-        require(transactionContext.ovmNUMBER == 0, "Only be callable at the start of a transaction");
+        // Prevent reentrancy to this function by requiring that the transaction context has not yet been
+        // initialized
+        require(transactionContext.ovmNUMBER == type(uint256).max, "Only callable as the start of a transaction");
```

Comment:
```suggestion
        require(transactionContext.ovmNUMBER == type(uint256).max, "OVM_ExecutionManager: Run can only be called at the start of a transaction.");
```

Reply:
Do we have a max line length? I've been feeling like we should. Generally I try not to go over 120.